lib/matplotlib/__init__.py

```diff
@@ -1292,7 +1292,7 @@ def __exit__(self, exc_type, exc_value, exc_tb):
         self.__fallback()
 
 
-def use(arg, warn=True, force=False):
+def use(arg, warn=False, force=True):
```

Comment:
I guess it's very remotely an API break in the sense that some previously invalid calls to use() would raise a warning and do nothing whereas they will now throw an exception, but I'm not sure if it's worth an API note :)

Reply:
API note added.  As per below I wonder if we really want to stick closer to the original behaviour and trap the Error, warn, and not switch the backend.